lib/bugsnag.ex

```diff
@@ -7,14 +7,20 @@ defmodule Bugsnag do
   @request_headers [{"Content-Type", "application/json"}]
 
   def start(_type, _args) do
-    config = Keyword.merge default_config, Application.get_all_env(:bugsnag)
+    config = default_config
+    |> Keyword.merge(Application.get_all_env(:bugsnag))
+    |> Enum.map(fn {k, v} -> {k, eval_config(v)} end)
 
-    if config[:use_logger] do
-      :error_logger.add_report_handler(Bugsnag.Logger)
+    case config[:use_logger] |> to_string do
+      "true" -> :error_logger.add_report_handler(Bugsnag.Logger)
```

Comment:
Is there a particular reason we need to cast this to a string, rather than matching on the boolean?